src/source_id/gen.sh

```diff
@@ -2,13 +2,18 @@
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-file="source_id.ml"
+file="generated.ml"
+
+if [ -n "$MOTOKO_RELEASE" ]
```

Comment:
I wonder if it would have been simpler to just us the ppx extension to include a version file rather than generate some source code, but no biggie. I guess this predates our embrace of ppx.